dashboard/lib/sample_data.rb

```diff
@@ -0,0 +1,181 @@
+include FactoryGirl::Syntax::Methods
+
+class SampleData
+  TEST_ACCOUNT_PASSWORD = "00secret".freeze
+  @@prng = nil
+
+  # Returns a seeded random number generator for consistent test data
+  def self.prng
+    @@prng ||= Random.new(0)
+    @@prng
+  end
+
+  # Creates sample data
+  def self.seed
+    raise "Should not be run outside of development" unless CDO.rack_env?(:development)
+
+    # Create a test teacher
+    teacher = create_teacher "testteacher@code.org", "TestTeacher Codeberg"
+
+    # Create normal-sized and large sections for each of CSF, CSD and CSP
+
+    create_section({teacher: teacher, name: "CSF 1",
+      login_type: Section::LOGIN_TYPE_PICTURE, grade: 2, age_min: 7,
+      age_max_inclusive: 9, script_name: "coursea-2018", num_students: 30,
+      use_imperfect_results: true}
+      )
```

Comment:
Ruby-style:
 1. The curly braces [aren't needed](https://github.com/rubocop-hq/ruby-style-guide#no-braces-opts-hash). For that matter neither are the parentheses, but they're helpful for multi-line declarations like this.
 2. The first arg should be on a new line and the final end parenthesis should be [unindented one level](https://github.com/rubocop-hq/ruby-style-guide#no-double-indent). I'm surprised Rubocop didn't complain about this.

```ruby
create_section(
  teacher: teacher, name: "CSF 1",
  login_type: Section::LOGIN_TYPE_PICTURE, grade: 2, age_min: 7,
  age_max_inclusive: 9, script_name: "coursea-2018", num_students: 30,
  use_imperfect_results: true
)
```

Minor Ruby-style nit: prefer [single quote strings](https://github.com/rubocop-hq/ruby-style-guide#strings) when not using interpolation.

I also tend to prefer splitting these long hashes into one key per line for readability. That's not necessary but it tends to make reading, comparing, and editing easier. This would be my ideal format:
```ruby
create_section(
  teacher: teacher, 
  name: 'CSF 1',
  login_type: Section::LOGIN_TYPE_PICTURE, 
  grade: 2, 
  age_min: 7,
  age_max_inclusive: 9, 
  script_name: 'coursea-2018', 
  num_students: 30,
  use_imperfect_results: true
)
```

Same for the other calls below

Reply:
Thx, all curly braces removed.

Reply:
Indentations fixed. I tried one line per parameter but it explodes that section of code into multiple screenfuls. I left it with multiple parameters per line because I can see the whole sample case generation strategy on one screen.